### ISSUE TITLE
Cache for perfomance tests

### ISSUE THREAD

**Issue by MathisFederico** [Newclid/Newclid#15]
We should have a way to store the cache for the github CI so tests don't take 1h to run @egretwAlker

**Comment by egretwAlker** [comment]
Yes there was a way in gitlab before. It should make sense that in GitHub we have something likewise. I'll also look into it later.

**Comment by egretwAlker** [comment]
I am wondering why we would want to have cache in CI though. We have regular tests for correctness. It would be nice to see the evaluation of performances over different commits, and with cache in CI we will no longer be able to see that.

But it's indeed a waste of resources as we have many commits, maybe we can configure so that we do performance on every last commit of a day? (Also important commits ofc with tags and releases)

**Comment by MathisFederico** [comment]
> But it's indeed a waste of resources as we have many commits, maybe we can configure so that we do performance on every last commit of a day? (Also important commits ofc with tags and releases)

Unit tests are for correctness, performance tests should only ensure no big performance drop is happening and I don't think building the cache is something we want to test for speed, and if so, we should only test it on smaller settings
